#! /bin/sh /usr/share/dpatch/dpatch-run
## asciidoc-8.4.dpatch by Jonas Fonseca <fonseca@diku.dk>
##
## DP: Fix build issues with asciidoc 8.4.
## DP:
## DP: This version seems to be stricter about presubs attributes.
## DP: Based on a patch by Ingmar Vanhassel.

@DPATCH@

diff a/tigrc.5.txt b/tigrc.5.txt
--- a/tigrc.5.txt
+++ b/tigrc.5.txt
@@ -9,11 +9,9 @@ tigrc - tig configuration file
 SYNOPSIS
 --------
 [verse]
-.............................................................................
 *set*   'variable' *=* 'value'
 *bind*  'keymap' 'key' 'action'
 *color* 'area' 'fgcolor' 'bgcolor' '[attributes]'
-.............................................................................
 
 
 DESCRIPTION
@@ -38,9 +36,7 @@ A few selective variables can be configured via the set command. The syntax
 is:
 
 [verse]
-..............................................................................
 *set* variables *=* value
-..............................................................................
 
 Examples:
 
@@ -146,9 +142,7 @@ Using bind commands keys can be mapped to an action when pressed in a given
 key map. The syntax is:
 
 [verse]
-..............................................................................
 *bind* 'keymap' 'key' 'action'
-..............................................................................
 
 Examples:
 
@@ -363,9 +357,7 @@ background combinations to certain areas. Optionally, an attribute can be
 given as the last parameter. The syntax is:
 
 [verse]
-..............................................................................
 *color* 'area' 'fgcolor' 'bgcolor' '[attributes]'
-..............................................................................
 
 Examples:
 
